app/assets/stylesheets/cafe_works.css.scss

```diff
@@ -0,0 +1,19 @@
+.cafe-date {
+  background: #eb7125;
```

Comment:
Color literals like `#eb7125` should only be used in variable declarations; they should be referred to via variable everywhere else.